aireview/src/aireview/domain/entities/review_comment.py

```diff
@@ -12,8 +12,8 @@ class ReviewComment:
     @staticmethod
     def from_json(data: Any) -> 'ReviewComment':
         return ReviewComment(
-            file_path=data.get("file_path", ""),
-            line=data.get("line", 0),
-            content=data.get("content", ""),
-            type=data.get("type", "")
+            file_path=data["file_path"],
```

Comment:
**style**: Removing dict.get() defaults makes the code less defensive. Consider keeping default values or adding explicit error handling for missing keys to prevent KeyError exceptions.